inventory_requests/serializers/RequestSerializer.py

```diff
@@ -8,29 +8,31 @@ class Meta:
         model = Item
         fields = ('id', 'name', 'quantity')
 
+
 class RequestSerializer(serializers.ModelSerializer):
     item = NestedItemSerializer(many=False, allow_null=False, read_only=True)
     item_id = serializers.IntegerField(required=True)
+
     class Meta:
         model = Request
         fields = ('id', 'owner', 'status', 'item', 'item_id', 'quantity', 'reason', 'timestamp', 'admin_timestamp', 'admin_comment', 'admin')
 
```

Comment:
In the future, we can make it write_only for the item_id

extra_kwargs = {'item_id': {'write_only': True}}